### PR TITLE
chore(happo): move stories with animations to cypress side

### PR DESCRIPTION
**Pull request by ozgurkececioglu**
[FX-2825]

### Description

As our [visual tests placing RFC](https://github.com/toptal/picasso/blob/master/docs/decisions/05-visual-tests-placing.md) points out that when we have flaky tests for snapshots, we should move them to the cypress side to eliminate flakiness and have more control. Stories for BarChart and CategoriesChart have animations and these animations can't be disabled globally. So, that's why they should be moved to Cypress side and disabled animations with prop.

BarChart component has some cypress tests already. So, I've just added happoSnapshot lines.

### How to test

- there won't be any BarChart or CategoriesChart snapshot on happo/storybook side anymore

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-2825]: https://toptal-core.atlassian.net/browse/FX-2825?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by separatio — APPROVED** [review]
Changes look good. Didn't manually test. Happo report looks :ok:

File: cypress/component/CategoriesChart.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+  },
+  with_active_engagement: {
+    team: {
+      with_active_engagement: 'Team With an Active Engagement Average'
+    },
+    user: {
+      with_active_engagement: 'Your With an Active Engagement'
+    }
+  }
+}
```
Comment (separatio) [inline cypress/component/CategoriesChart.spec.tsx:225]: Same as above. Just move all the data in a separate file and import it in the test.

File: .changeset/eleven-eels-tie.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+'@topkit/analytics-charts': minor
+---
+
+---
+### CategoriesChart
+
+- added `isAnimationActive` prop to disable animations for the chart
```
Comment (separatio) [inline .changeset/eleven-eels-tie.md:8]: What do you think of having the prop called simply `animate`? Or `animations`?
If I understood correctly, it can be true/false and enable/disable animations.

Comment (ozgurkececioglu) [reply .changeset/eleven-eels-tie.md:8]: We have the same for [`BarChart`](https://github.com//toptal/picasso/blob/7470243feefc9a84170d03cb6dde1eac3acbea0e/packages/picasso-charts/src/BarChart/BarChart.tsx#L65) and it comes from recharts actually. I have just used the same. Do you mean to change both of them?

Comment (separatio) [reply .changeset/eleven-eels-tie.md:8]: No. If we have them already, this is :ok:

File: packages/topkit-analytics-charts/src/CategoriesChart/CategoriesChart.tsx [hunk 3]
```diff
@@ -5,7 +5,10 @@ import { DataItem, LabelMap, TooltipMap } from './types'
 import { formatData, getColor } from './utils'
 import CategoriesChartTooltip from '../CategoriesChartTooltip'
 
-export type Props = Pick<BarChartProps<string>, 'width' | 'height'> & {
+export type Props = Pick<
+  BarChartProps<string>,
+  'width' | 'height' | 'isAnimationActive'
+> & {
```
Comment (OndrejTuma) [inline packages/topkit-analytics-charts/src/CategoriesChart/CategoriesChart.tsx:11]: Can you please explain to me what this does?

Comment (OndrejTuma) [reply packages/topkit-analytics-charts/src/CategoriesChart/CategoriesChart.tsx:11]: Oh never mind, I got it 🙂

File: cypress/component/BarChart.spec.tsx [hunk 4]
```diff
@@ -108,6 +71,8 @@ describe('BarChart', () => {
   it('shows default tooltip on hover', () => {
     mount(<TestBarChart />)
 
+    cy.get('body').happoScreenshot()
```
Comment (OndrejTuma) [inline cypress/component/BarChart.spec.tsx:74]: Shouldnt this screenshot be rather captured in storybook? 🤔

Comment (ozgurkececioglu) [reply cypress/component/BarChart.spec.tsx:74]: If you check our fresh [visual tests placing](https://github.com/toptal/picasso/blob/e50eda87ac3072410d64beccf12f3286a7f1a501/docs/decisions/05-visual-tests-placing.md) RFC, you will see that it is our official way to handle stories with animations now.

**Review by OndrejTuma — APPROVED** [review]
Some nits, but those can't outshine the good work you've done 👍

File: .github/workflows/davinci-alpha-package.yml [hunk 5]
```diff
@@ -50,7 +50,7 @@ jobs:
         uses: actions/checkout@v2
         with:
           repository: toptal/davinci-github-actions
-          ref: v2
+          ref: v2.2.0
```
[the remark this answers is not in the document]
Comment (ozgurkececioglu) [reply .github/workflows/davinci-alpha-package.yml:53]: Happy to 🙂

**Review by separatio — APPROVED** [review]
Alright. Things look good. :+1:

**Review by LashaJini — APPROVED** [review]
Tested locally and no issues so far :+1:

File: .github/workflows/davinci-alpha-package.yml [hunk 6]
```diff
@@ -50,7 +50,7 @@ jobs:
         uses: actions/checkout@v2
         with:
           repository: toptal/davinci-github-actions
-          ref: v2
+          ref: v2.2.1
```
Comment (OleksandrNechai) [inline .github/workflows/davinci-alpha-package.yml:53]: How is this change related to the topic of PR?

**Review by OleksandrNechai — APPROVED** [review]
LGTM 👍

File: .github/workflows/davinci-alpha-package.yml (hunk 6 above)
Comment (TomasSlama) [reply .github/workflows/davinci-alpha-package.yml:53]: @OleksandrNechai I think CI were failing because of `yarn install gha`

**Review by TomasSlama — APPROVED** [review]

File: .github/workflows/davinci-alpha-package.yml (hunk 6 above)
Comment (ozgurkececioglu) [reply .github/workflows/davinci-alpha-package.yml:53]: It was affecting the CI so I had to add it
